client/src/components/SoundPanel.tsx

```diff
@@ -0,0 +1,131 @@
+import gongIcon from '../assets/instruments/gong.png';
+import harpIcon from '../assets/instruments/harp.png';
+// Funny
+import gotemIcon from '../assets/funny/gottem.png';
+import noiceIcon from '../assets/funny/noice.png';
+import stop_itIcon from '../assets/funny/stop_it.png';
+import ahhhIcon from '../assets/funny/ahhh.png';
+import airIcon from '../assets/funny/air.png';
+import applauseIcon from '../assets/funny/applause.png';
+import clangIcon from '../assets/funny/clang.png';
+import groanIcon from '../assets/funny/groan.png';
+import hornIcon from '../assets/funny/horn.png';
+import laughIcon from '../assets/funny/laugh.png';
+// Nature
+import beeIcon from '../assets/nature/bee.png';
+import dogIcon from '../assets/nature/dog.png';
+import flying_foxIcon from '../assets/nature/flying_fox.png';
+import lightningIcon from '../assets/nature/lightning.png';
+import natureIcon from '../assets/nature/nature.png';
+import sealionIcon from '../assets/nature/sealion.png';
+import waterIcon from '../assets/nature/water.png';
+
+const soundCategories: ISoundCategories[] = [
+	{ category: 'Instruments', isActive: true, id: uuidv4() },
+	{ category: 'Funny', isActive: false, id: uuidv4() },
+	{ category: 'Nature', isActive: false, id: uuidv4() }
+];
+
+const soundList: ISoundPairs[] = [
+	// Instruments
+	{ icon: drumIcon, type: 'drum', category: 'Instruments' },
+	{ icon: cymballIcon, type: 'cymbal', category: 'Instruments' },
+	{ icon: guitarIcon, type: 'guitar', category: 'Instruments' },
+	{ icon: trumpet, type: 'trumpet', category: 'Instruments' },
+	{ icon: gongIcon, type: 'gong', category: 'Instruments' },
+	{ icon: harpIcon, type: 'harp', category: 'Instruments' },
+	// Funny
+	{ icon: gotemIcon, type: 'meme', category: 'Funny' },
+	{ icon: stop_itIcon, type: 'stop_it', category: 'Funny' },
+	{ icon: noiceIcon, type: 'noice', category: 'Funny' },
+	{ icon: ahhhIcon, type: 'ahh', category: 'Funny' },
+	{ icon: airIcon, type: 'air', category: 'Funny' },
+	{ icon: applauseIcon, type: 'applause', category: 'Funny' },
+	{ icon: clangIcon, type: 'clang', category: 'Funny' },
+	{ icon: groanIcon, type: 'groan', category: 'Funny' },
+	{ icon: hornIcon, type: 'horn', category: 'Funny' },
+	{ icon: laughIcon, type: 'laugh', category: 'Funny' },
+	// Nature
+	{ icon: beeIcon, type: 'bee', category: 'Nature' },
+	{ icon: dogIcon, type: 'dog', category: 'Nature' },
+	{ icon: flying_foxIcon, type: 'flying_fox', category: 'Nature' },
+	{ icon: lightningIcon, type: 'lightning', category: 'Nature' },
+	{ icon: natureIcon, type: 'nature', category: 'Nature' },
+	{ icon: sealionIcon, type: 'sealion', category: 'Nature' },
+	{ icon: waterIcon, type: 'water', category: 'Nature' }
+];
+
+interface ISoundPanelProps {
+	sendSound: (soundText: string, soundType: string) => void;
+}
+
+function SoundPanel({ sendSound }: ISoundPanelProps) {
+	const [activeCategories, setActiveCategories] = useState(soundCategories);
```

Comment:
there's a better way to do a lot of this logic, we don't need to load the entire soundCategories as a state because the only thing that changes is the category which is a string, like "Instrument" or "Nature", therefore we only need to useState for a string

please review the changes on your repo on the branch 'simpler-soundpanel-logic', the main points:

no need to keep entire categories in state, only track what changes
enums / types can create cleaner simpler code